engine/src/main/java/org/hibernate/search/bridge/spi/FieldType.java

```diff
@@ -48,5 +48,13 @@
 	/**
 	 * A double field.
 	 */
-	DOUBLE
+	DOUBLE,
+
+	/**
+	 * An object field, a generic object that doesn't have a unique definition in all the backends.
```

Comment:
The "object" being repeated is confusing. "backends" at the end of the sentence is as well.

alternative proposal:
"A composite object that will benefit from different encoding on each indexing technology."